src/redux/books/books.js

```diff
@@ -1,6 +1,19 @@
 const ADD_BOOK = 'react_book_store/books/ADD_BOOK';
 const REMOVE_BOOK = 'react_book_store/books/REMOVE_BOOK';
-const DEFAULT_STATE = [];
```

Comment:
- [ ] You have successfully implemented redux store 💯 but you didn't implement remove functionality 😨 Kindly implement it because user is not able to remove book 🤔